Source/OpenInExternalBrowserView.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  OpenInExternalBrowserView.swift
+//  edX
+//
+//  Created by Saeed Bashir on 4/21/21.
+//  Copyright © 2021 edX. All rights reserved.
+//
+
+import Foundation
+
+protocol OpenInExternalBrowserViewDelegate: class {
+    func openInExternalBrower()
+}
+
+class OpenInExternalBrowserView: UIView, UITextViewDelegate {
+    let container = UIView()
+    let messageLabel = UILabel()
+    let button = UIButton()
+
+    weak var delegate: OpenInExternalBrowserViewDelegate?
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+
+        setupView()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func setupView() {
+
+        messageLabel.numberOfLines = 0
+        container.backgroundColor = OEXStyles.shared().infoXXLight()
+
+        container.addSubview(messageLabel)
+        container.addSubview(button)
+        addSubview(container)
+
+        container.snp.remakeConstraints { make in
+            make.edges.equalTo(self)
+        }
+
+        messageLabel.snp.remakeConstraints { make in
+            make.leading.equalTo(container).offset(StandardHorizontalMargin)
+            make.trailing.equalTo(container).inset(StandardHorizontalMargin)
+            make.centerY.equalTo(container)
+        }
+
+        button.snp.remakeConstraints{ make in
+            make.edges.equalTo(container)
+        }
+
+        let textStyle : OEXTextStyle = OEXTextStyle(weight : .normal, size: .small, color: OEXStyles.shared().neutralXDark())
```

Comment:
no need to specify `OEXTextStyle` type

```suggestion
        let textStyle = OEXTextStyle(weight: .normal, size: .small, color: OEXStyles.shared().neutralXDark())
```